libtransmission/port-forwarding-upnp.cc

```diff
@@ -350,18 +366,23 @@ tr_port_forwarding_state tr_upnpPulse(tr_upnp* handle, tr_port port, bool is_ena
             fmt::arg("url", handle->urls.controlURL),
             fmt::arg("type", handle->data.first.servicetype),
             fmt::arg("address", handle->lanaddr),
-            fmt::arg("port", port.host())));
+            fmt::arg("port", local_port.host())));
 
         if (handle->isMapped)
         {
-            tr_logAddInfo(fmt::format(_("Port {port} is forwarded"), fmt::arg("port", port.host())));
-            handle->port = port;
+            tr_logAddInfo(fmt::format(
+                _("Port forwarded local port {local_port} to {advertised_port}"),
```

Comment:
nit: `Port` in the beginning of log message seems redundant

Reply:
7fe91ea